R/tags.R

```diff
@@ -698,7 +713,35 @@ findDependencies <- function(tags, tagify = TRUE) {
     }
   }
   childDeps <- unlist(lapply(children, findDependencies, tagify = FALSE), recursive = FALSE)
-  c(childDeps, if (!is.null(dep)) dep)
+  c(childDeps, if (!is.null(deps)) deps)
+}
+
+resolveFunctionalDependencies <- function(deps) {
+  if (!length(deps)) {
+    return(deps)
+  }
+  deps <- lapply(deps, function(x) {
+    if (is_html_dependency(x)) {
+      return(list(x))
```

Comment:
Why is this wrapped in a list? Same for below, where it returns `list(deps)`.

Reply:
Since it's possible for `x()` to return a list of dependencies, I think it's easiest to just always return a list of dependencies so we can do `unlist(deps, recursive = FALSE)`

Reply:
Makes sense.